rtl/src/main/scala/tensil/PlatformConfig.scala

```diff
@@ -3,21 +3,23 @@
 
 package tensil
 
-import tensil.mem.MemKind.{MemKind, RegisterBank}
+import tensil.mem.MemKind.{Type, RegisterBank}
```

Comment:
Could we change `Type` to `Kind` or something similar? When I see a language keyword used as a user class/variable I get very confused.